proj/src/main/webapp/style.css

```diff
@@ -298,4 +310,79 @@ footer a:hover {
   font-size:2rem;
   font-weight: normal;
   color: #FFFFFF;
+}
+
+
+
+/* chatroom left navigation */
```

Comment:
I thought these styles were split into a separate CSS file for chat stuff.

Reply:
yup this will be cleaned on the cleaning branch